k8s/kube-base/deployment-xdmod.yaml

```diff
@@ -42,6 +42,34 @@ spec:
               mountPath: /run/httpd
             - name: vol-php-session
               mountPath: /var/lib/php/session
+        - image: moc-xdmod:latest
+          name: xdmod-acl-work-a-round
+          imagePullPolicy: IfNotPresent
+          command: ["/usr/bin/bash"]
+          args: ["-c", "while true; do /usr/bin/acl-config; sleep 420; done"]
+          ports:
```

Comment:
The `ports` section here might be problematic, since you've already indicated that the `xdmod` container is using port 8080 (and we're not exposing a network service in this container in any case).